tests/tests_suite_test.go

```diff
@@ -318,6 +320,9 @@ func createApp(name string, options ...string) *Session {
 	Expect(err).NotTo(HaveOccurred())
 	Eventually(cmd).Should(Say("created %s", name))
 	Eventually(cmd).Should(Exit(0))
+	Eventually(cmd).Should(SatisfyAll(
```

Comment:
These need to move out of this function and back into the tests invoking this function, as validations on the output aren't always the same.  For instance, if consider the following `deis apps:create <appName> --no-remote` case (No `"Git remote deis added"` will be seen:
```
$ deis apps:create test-699315197 --no-remote
Creating Application... done, created test-699315197
remote available at ssh://git@deis.130.211.185.40.xip.io:2222/test-699315197.git
$ deis apps:destroy --app=test-699315197 --confirm=test-699315197
Destroying test-699315197...
done in 0s
$ rm -rf .git

• Failure in Spec Setup (BeforeEach) [2.860 seconds]
Builds
/Users/vaughndice/work/go/src/github.com/deis/workflow-e2e/tests/builds_test.go:130
  with a logged-in user
  /Users/vaughndice/work/go/src/github.com/deis/workflow-e2e/tests/builds_test.go:129
    with a deployed app
    /Users/vaughndice/work/go/src/github.com/deis/workflow-e2e/tests/builds_test.go:128
      can create a build from an existing image ("deis pull") [BeforeEach]
      /Users/vaughndice/work/go/src/github.com/deis/workflow-e2e/tests/builds_test.go:127

      No future change is possible.  Bailing out early after 0.000s.
      Got stuck at:

          remote available at ssh://git@deis.130.211.185.40.xip.io:2222/test-699315197.git

      Waiting for:
          Git remote deis added

      /Users/vaughndice/work/go/src/github.com/deis/workflow-e2e/tests/tests_suite_test.go:325
------------------------------
```